css/checkout-page.css

```diff
@@ -0,0 +1,76 @@
+	.dialog-box {
```

Comment:
Is this for the modal? If so, let's remove the file and not enqueue it. Any CSS needed can go into frontend.css. Let's have one frontend css file only.

Reply:
Removing